packages/flutter/lib/src/material/search.dart

```diff
@@ -188,6 +188,44 @@ abstract class SearchDelegate<T> {
   ///  * [AppBar.leading], the intended use for the return value of this method.
   Widget? buildLeading(BuildContext context);
 
+  /// A flag to automatically control the visibility of the leading widget in the [AppBar].
+  ///
+  /// When set to `true`, the [AppBar] will automatically imply a leading widget. This is typically
+  /// a [BackButton] when there's a previous route in the navigation stack, or an [IconButton]
+  /// with a hamburger menu icon when there's not.
+  ///
+  /// When set to `false`, the [AppBar] will not imply a leading widget. In this case, the space
+  /// reserved for the leading widget will be empty, unless [buildLeading] is overridden
+  /// to provide a custom leading widget.
+  ///
+  /// When set to `null`, the [AppBar] will defer to the platform's default behavior.
+  ///
+  /// See also:
+  ///
+  ///  * [AppBar.automaticallyImplyLeading], which is controlled by this property.
+  bool? automaticallyImplyLeading;
```

Comment:
Would `/// {@macro flutter.material.appbar.automaticallyImplyLeading}` be sufficient here? What do you think?

Reply:
yep, of course. i've updated this line. thank you